services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchPropertiesSpec.groovy

```diff
@@ -28,10 +28,7 @@ class SwitchPropertiesSpec extends HealthCheckSpecification {
         when: "Update switch properties"
         SwitchPropertiesDto switchProperties = new SwitchPropertiesDto()
         def newMultiTable = !initSwitchProperties.multiTable
-        def newTransitEncapsulation = (initSwitchProperties.supportedTransitEncapsulation.size() == 1) ?
-                [FlowEncapsulationType.TRANSIT_VLAN.toString().toLowerCase(),
-                 FlowEncapsulationType.VXLAN.toString().toLowerCase()].sort() :
-                [FlowEncapsulationType.VXLAN.toString().toLowerCase()]
+        def newTransitEncapsulation = [FlowEncapsulationType.TRANSIT_VLAN.toString().toLowerCase()]
```

Comment:
As for would be better to change the procedure of selecting switch for manipulation
`def sw = topology.activeSwitches.find { it.features.contains(SwitchFeature.NOVIFLOW_COPY_FIELD) }`

otherwise we can't be sure that we are really able to update switch properties.